src/main/java/io/antmedia/streamsource/StreamFetcherManager.java

```diff
@@ -68,10 +113,12 @@ public Result startStreaming(Broadcast broadcast) {
 		Result result=new Result(false);
 
 		try {
-			StreamFetcher streamScheduler = new StreamFetcher(broadcast,scope);
+			StreamFetcher streamScheduler = streamFetcherFactory.make(broadcast, scope);
```

Comment:
instead of making StreamFetcherFactory a static class, make "make" method a static method, so there will be no need to create an instance and keep as variable.

Reply:
Making make method static is ok. On the other hand, in testing we give a mock StreamFetcherFactory instance as a parameter to create a test case. How can we do that not using an instance?

Reply:
I see. Please just take a look at here
https://github.com/ant-media/Ant-Media-Server/blob/dev/src/test/java/io/antmedia/test/StreamFetcherUnitTest.java#L258

We give a different mock StreamFactory class. If we call StreamFactory.make static function directly how we give a separate mock in the test case above? Maybe we need to change some other things in the code as well.